app/models/sales_log.rb

```diff
@@ -219,6 +219,12 @@ def old_persons_shared_ownership?
     type == 24
   end
 
+  def is_bedsit?
+    return if proptype.nil?
```

Comment:
I don't think this is necessary - if proptype is nil, then 225 returns nil == 2 => false anyway